content/guides/4.connect/2.filter-rules.md

```diff
@@ -76,39 +76,60 @@ This filter checks the `title` field contains the case-sensitive substring 'Dire
 
 ## Relational Fields
 
-You can specify related values by nesting field names. For example, if you have a relational Many to One `author` field, and want to specify a filter on the author's `name` field:
+You can filter items based on related data by nesting field names in your query. This allows you to query items not just by their own fields, but also by values in related collections.
 
+### Many-to-One
+
+You can filter items with Many-to-One relations by specifying values of their respective fields.
+
+::callout{icon="material-symbols:info-outline"}
+**Example**<br/>
+
+For example, if you have an articles collection with a relational Many-to-One `author` field, you can filter articles based on the author's details—such as their `name`.
 ```json
 {
-  "authors": {
-    "authors_id": {
+  "author": {
       "name": {
         "_eq": "Rijk van Zanten"
       }
-    }
   }
 }
 ```
+::
+
+### One-to-Many and Many-to-Many
+
+The `_some` and `_none` filter operators are used for filtering One-to-Many and Many-to-Many relational fields in API queries. These allow you to filter items based on conditions applied to their related collections.
 
-When applying filters to a One to Many field, Directus will default to a 'some' search.
+The `_some` operator matches items where at least one related item meets the condition.
+By default, Directus will apply the `_some` operator when querying One-to-Many and Many-to-Many relational queries.
 
 ::callout{icon="material-symbols:info-outline"}
-**Example**  
-This filter checks all related category names and will check is at least one value is equal to 'Recipe':
+**Example**<br/>
+
+This filter matches all items where at least one related category has the name "Recipe":
+
 ```json
 {
   "categories": {
-    "name": {
-      "_eq": "Recipe"
+    "_some": {
+      "name": {
+        "_eq": "Recipe"
+      }
     }
   }
 }
 ```
+
+In the above, `categories` is the relational field in the collection. `_some` checks if at least one related category has the name "Recipe". If an article has one or more categories named "Recipe", it will be included in the result.
```

Comment:
Since we are talking about records all the time

```suggestion
In the above, `categories` is the relational field in the collection. `_some` checks if at least one related category has the name "Recipe". If a record has one or more categories named "Recipe", it will be included in the result.
```